astropy/coordinates/angles/utils.py

```diff
@@ -4,14 +4,7 @@
 used internally in astropy.coordinates.angles, and of possible.
```

Comment:
While you are at it, could you complete the docstring? Not sure how it ended up broken up...  I guess it would be "and of possible use externally."

Reply:
✅ Done!